api/redis/processReminders.js

```diff
@@ -0,0 +1,71 @@
+const reminderStore = require('./client.js').reminders;
+const bot = require('../bot.js');
+
+const scanReminderStore = () => {
+  return reminderStore.scanAsync(0).then((data) => {
```

Comment:
whats the `0` here?

Reply:
`scan` takes a cursor argument, as described [here](https://redis.io/commands/scan)